src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/StatusRoutesSpec.scala

```diff
@@ -23,6 +23,14 @@ import scala.concurrent.duration._
 class StatusRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest with CommonTestData with TestLeoRoutes with TestComponent {
   override implicit val patienceConfig = PatienceConfig(timeout = 1.second)
 
+  "GET /version" should "give 200 for ok" in {
+    eventually {
+      Get("/version") ~> leoRoutes.route ~> check {
+        status shouldEqual StatusCodes.OK
```

Comment:
Could the test validate the actual version string returned? Does it return something for tests or `n/a`?